Resources/Textures/_DV/Clothing/Head/Hats/beret_atmospherics.rsi/meta.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e. Modified by AirlineFood",
```

Comment:
Find a reference to the original `.dmi` file these assets, and copy the permalink here